lib/v2/untag/category.js

```diff
@@ -0,0 +1,65 @@
+const got = require('@/utils/got');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+module.exports = async (ctx) => {
+    const category = ctx.params.category ?? 'method';
+    const limit = ctx.query.limit ? parseInt(ctx.query.limit) : 20;
+
+    const rootUrl = 'https://utgd.net';
+    const apiUrl = `${rootUrl}/api/v2/pages`;
+    const currentUrl = `${rootUrl}/category/s/${category}`;
+    const slugUrl = `${rootUrl}/api/v2/category/slug/${category}`;
+
+    let response = await got({
+        method: 'get',
+        url: slugUrl,
+    });
+
+    const data = response.data;
+
+    response = await got({
+        method: 'get',
+        url: apiUrl,
+        searchParams: {
+            type: 'article.Article',
+            fields: `article_category(category_name),article_tag(tag_name),title,article_image,article_author,article_content,article_published_time`,
+            article_category: data.id,
+            order: '-article_published_time',
+            limit,
+        },
+    });
+
+    const items = await Promise.all(
+        response.data.items.map((item) =>
+            ctx.cache.tryGet(`untag-${item.id}`, async () => {
+                const authorResponse = await got({
+                    method: 'get',
+                    url: `${rootUrl}/api/v2/user/profile/${item.article_author.id}`,
```

Comment:
```suggestion
                    url: `${rootUrl}/api/v2/user/profile/${item.article_author.id}/`,
```
https://github.com/DIYgod/RSSHub/actions/runs/5105246772/jobs/9176711212#step:12:40